packages/oui-icons/icons.less

```diff
@@ -13,4 +14,12 @@
   &_bicolor {
     #oui > .icon-bicolor();
   }
+
+  &_animate {
+    &__shake {
```

Comment:
Not BEM compliant. We could go with something like `oui-icon_shake`, since It's a modifier for `oui-icon`.